### PR TITLE
Add address property

### PR DESCRIPTION
**Pull request by ilikecake**
Added a read-only address property to the class. This might be helpful to identify sensors if you have multiple on the bus.

### CODE REVIEW

File: adafruit_ds18x20.py [hunk 1]
```diff
@@ -155,3 +155,11 @@ def read_temperature(self) -> float:
         """Read the temperature. No polling of the conversion busy bit
         (assumes that the conversion has completed)."""
         return self._read_temp()
+
+    @property
+    def address(self):
+        """Returns the address of the device. Might be useful if there are multiple sensors on
+        the bus.
```
Comment (tannewt) [inline adafruit_ds18x20.py]:
```suggestion
        """The address of the device. Useful if there are multiple sensors on the bus.
```

**Review by tannewt — CHANGES_REQUESTED** [review]
Thanks! One suggestion.

**Comment by ilikecake** [comment]
I have updated the code with the requested change. Do I need to do something to resolve the change request?

**Review by FoamyGuy — APPROVED** [review]
Looks good, requested change was handled in the latest commit.

Thank you @ilikecake!

**Pull request merged**